### DISCUSSION TITLE
Question to the 2.0 roadmap

### DISCUSSION THREAD

**Discussion by etkaar** [jedisct1/libsodium#1408] [General]
In the roadmap for libsodium 2.0 it says:

`Switch to a new API (libhydrogen/WASI-crypto)`

What does that mean? libsodium will become libhydrogen? Aren't these two different librarys with different goals?

I ask because I highly rely on libsodium 1.0 in Python and PHP and did not plan to replace it, since I finished my implementations.

**Comment by jedisct1** [comment]
Don't worry, your project won't stop working.

**Comment by etkaar** [comment]
Alright. But what does the quote mean?
